wajibisha/templates/about.html

```diff
@@ -16,21 +16,42 @@ <h2>About</h2>
-          mentitum moderatius cu. Sit munere facilis accusam eu dicat falli consulatu at vis. Te facilisis mnesarchum qui posse omnium mediocritatem est cu. Modus argumentum ne qui tation efficiendi in eos. Ei mea falli legere efficiantur et tollit aliquip debitis mei. No deserunt mediocritatem mel. Lorem ipsum dolor sit amet et delectus accommodare his consul copiosae legendos at vix ad putent delectus delicata usu.<br>
-          Vidit dissentiet eos cu eum an brute copiosae hendrerit. Eos erant dolorum an. Per facer affert ut. Mei iisque mentitum moderatius cu. Sit munere facilis accusam eu dicat falli consulatu at vis. Te facilisis mnesarchum qui posse omnium mediocritatem est cu. Modus argumentum ne qui tation efficiendi in eos. Ei mea falli legere efficiantur et tollit aliquip debitis mei.<br>
-          No deserunt mediocritatem mel. Lorem ipsum dolor sit amet et delectus accommodare his consul copiosae legendos at
-          vix ad putent delectus delicata usu. Vidit dissentiet eos cu eum an brute copiosae hendrerit. Eos erant dolorum an. Per facer affert ut. Mei iisque mentitum moderatius cu. Sit munere facilis accusam eu dicat falli consulatu at vis. Te facilisis mnesarchum qui posse omnium mediocritatem est cu. Modus argumentum ne qui tation efficiendi in eos. Ei mea falli legere efficiantur et tollit aliquip debitis mei. No deserunt mediocritatem mel. Lorem ipsum dolor sit amet et delectus accommodare his consul copiosae legendos at vix ad putent delectus delicata usu. Vidit dissentiet eos cu eum an brute copiosae hendrerit. Eos erant dolorum an. Per facer affert ut. Mei iisque mentitum moderatius cu. Sit munere facilis accusam eu dicat falli consulatu at vis. Te facilisis mnesarchum qui posse omnium mediocritatem est cu. Modus</p>
-          
+    <div class="body-text wrapper clearfix text-center pl-5 pr-5 pb-5">
+      <h4 style="color:#0067A3;">CONTACT US</h4>
+      <br/>
+      <div class="d-flex flex-row justify-content-center pb-5">
+        <div class="col-5 text-left pb-5 pl-5 offset-1">
+          PesaCheck HQ:<br>
+          Nairobi Garage, 8th Floor, Pinetree Plaza, <br>
+          Kaburu Drive, Kilimani, Nairobi, Kenya.<br>
+
+          <p>
+            Phone: +254 727 588456<br>
+            Email: hello@pesacheck.org
+          </p>
+        </div>
+
+        <div class="col-4 text-left pb-5">
+          PesaCheck Tanzania:<br>
+          Bongo Live, No 6, Zanaki Street,<br>
+          Dar es Salaam, Tanzania.<br>
+
+          <p>tanzania@pesacheck.orgs</p>
+        </div>
+
+        <div class="col-4 text-left pl-5 pb-5">
+          PesaCheck Uganda:<br>
+          Design Hub, Plot 3, Bata Close,<br>
+          5th Street, Kampala, Uganda.<br>
+
+          <p>uganda@pesacheck.org</p>
+        </div>
       </div>
     </div>
-    {% include 'blocks/team.html' %}
-
+  </div>
     {% endblock %}
 </div>
```

Comment:
This about page looks like it's a copy of what is on Medium. Should we not point to that instead if nothing different?